codex-rs/core/src/exec.rs

```diff
@@ -29,6 +31,7 @@ const DEFAULT_TIMEOUT_MS: u64 = 10_000;
 /// Hardcode this since it does not seem worth including the libc craate just
```

Comment:
You can fix my typo while here :P

```suggestion
/// Hardcode this since it does not seem worth including the libc crate just
```

Reply:
s/for this/for these/?

Reply:
ae7ab970e057b72426b1049e0c87ee06c3a17354